getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -25,9 +25,11 @@ To create a Telerik-enabled Blazor project:
 
 1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.
 
-## Creating the Project
+## Using the Wizard
 
-The **Create New Project** wizard provides two options to start you project:
+### Section 1: Choose how to start the project
```

Comment:
Looking at the entire thing, it seems to me we might be better off using Steps instead. Sections made sense when there was the possibility to combine multiple steps into one (which is no longer the case because of the way the wizard is structured now).

```suggestion
### Step 1: Choose How to Start
```